share/zephyr-package/cmake/zephyr_package_search.cmake

```
@@ -49,10 +49,14 @@ endmacro()
 # - VERSION_CHECK              : This is the version check stage by CMake find package
 # - CANDIDATES_PREFERENCE_LIST : List of candidate to be preferred, if installed
 macro(check_zephyr_package)
-  set(options CHECK_ONLY SEARCH_PARENTS VERSION_CHECK)
-  set(single_args WORKSPACE_DIR ZEPHYR_BASE)
-  set(list_args CANDIDATES_PREFERENCE_LIST)
-  cmake_parse_arguments(CHECK_ZEPHYR_PACKAGE "${options}" "${single_args}" "${list_args}" ${ARGN})
+  set(check_zephyr_package_options CHECK_ONLY SEARCH_PARENTS VERSION_CHECK)
+  set(check_zephyr_package_single_args WORKSPACE_DIR ZEPHYR_BASE)
+  set(check_zephyr_package_list_args CANDIDATES_PREFERENCE_LIST)
+  cmake_parse_arguments(CHECK_ZEPHYR_PACKAGE
+    "${check_zephyr_package_options}"
+    "${check_zephyr_package_single_args}"
+    "${check_zephyr_package_list_args}"
+    ${ARGN})
```

The variables aren't really needed in the first place, so you could also do:
```suggestion
  cmake_parse_arguments(CHECK_ZEPHYR_PACKAGE
    "CHECK_ONLY;SEARCH_PARENTS;VERSION_CHECK" # options
    "WORKSPACE_DIR;ZEPHYR_BASE" # single args
    "CANDIDATES_PREFERENCE_LIST" # multi args
    ${ARGN})
```